composer.json

```diff
@@ -11,9 +11,9 @@
     }],
     "require": {
         "php": ">=5.4",
-        "symfony/css-selector": "2.8.*",
+        "apache/log4php": "2.3.0",
         "facebook/php-sdk-v4": "~5.0",
-        "apache/log4php": "2.3.0"
+        "symfony/css-selector": "^3.0.4"
```

Comment:
Hello @localheinz, thanks for your PR!

Why have you moved symfony/css-selector from 2.8.* to 3.0.4?
`"symfony/css-selector": "^3.0.4`
We downgraded it just because 2.8.* is compatible with PHP 5.4

The order of packages makes sense, but we are not intended in not being compatible with PHP 5.4 because of its usage nowadays.

Can you change the version back to 2.8.*?